api/useQuerySchemas.ts

```diff
@@ -8,7 +8,7 @@ const DIGITAL_PLANNING_DATA_SCHEMAS_JSON_URL =
 
 // TODO: type the version more strictly
 const useQuerySchemas = (
-  version: string,
+  schemaName: string,
```

Comment:
Renaming only: `version` was semantically wrong here. Our top-level toggle is based on schema name (eg preapp, application, etc) and all loaded schemas currently default to `@next` version